examples/games/alien_cake_addict.rs

```diff
@@ -9,9 +10,23 @@ enum GameState {
     GameOver,
 }
 
+#[derive(Resource)]
+struct SpawnTimer(Timer);
+
```

Comment:
This run criteria, in combination with the system set, feels like a lot of boilerplate for a behavior that is specific to the `spawn_bonus` behavior. In cases like this I think we should encourage the pattern of just checking the timer in the `spawn_bonus` system.

Reply:
I think moving the check into the `spawn_bonus` system makes sense to simplify this example, but a condition/criteria is the best practice since we'll avoid spawning tasks when we don't need to.

The problem atm is we can't give `spawn_bonus` its own criteria and put it in the `Playing` state's `Update` system set, but we can clean that up once we've brought in all the stageless api changes.